assets/css/settings/_chess.scss

```diff
@@ -0,0 +1,2 @@
+$colours: "black" "white";
```

Comment:
`$colours` feels quite generically named. Could/should it be more specific/descriptive?